pyproject.toml

```diff
@@ -82,3 +160,70 @@ log_format = "%(asctime)s.%(msecs)03d %(levelname)-8s %(name)-12s %(filename)s:%
 log_date_format = "%Y-%m-%dT%H:%M:%S"
 log_cli_format = "%(asctime)s.%(msecs)03d %(levelname)-8s %(name)-12s %(filename)s:%(lineno)s:%(funcName)s %(message)s"
 log_cli_date_format = "%Y-%m-%dT%H:%M:%S"
+
+[tool.isort]
+profile = "black"
+py_version = 37
+line_length = 88
+src_paths= ["rdflib", "test"]
+supported_extensions = ["pyw", "pyi", "py"]
+skip = [
+    '.eggs',         # exclude a few common directories in the
+    '.git',          # root of the project
+    '.hg',
+    '.mypy_cache',
+    '.pytest_cache',
+    '.tox',
+    '.venv',
+    '.var',
+    '.github',
+    '_build',
+    'htmlcov',
+    'benchmarks',
+    'examples',       # No need to Black examples
```

Comment:
minor: I think this comment may be misleading, since this is config for isort rather than Black.

Reply:
Sounds reasonable to me - though we inherited that wording from the config that was in `setup.cfg`; we simply moved it to `pyproject.toml` here. I'll let @aucampia comment on whether we should update the wording here to say "No need to **lint** example", or do that in another PR.

Reply:
I changed it now on the PR branch, though actually we should probably be blacking and linting examples, but that can be sorted out in future PRs.